internal/deviations/deviations.go

```diff
@@ -515,6 +515,12 @@ func NtpNonDefaultVrfUnsupported(dut *ondatra.DUTDevice) bool {
 	return lookupDUTDeviations(dut).GetNtpNonDefaultVrfUnsupported()
 }
 
+// MatchedPacketsOctetsUnsupported returns if telemetry path /qos/interfaces/interface/input/classifiers/classifier/terms/term/state/matched-packets and matched-octets is not supported.
+func MatchedPacketsOctetsUnsupported(dut *ondatra.DUTDevice) bool {
```

Comment:
New deviations should be added to the metadata proto, not as flags. Please follow the instructions here:
https://github.com/openconfig/featureprofiles/blob/main/internal/deviations/README.md

Reply:
I have added deviations in metadata proto, are there any other changes required ?